src/bin/metadata.rs

```diff
@@ -0,0 +1,66 @@
+use cargo::ops::{output_metadata, OutputTo, OutputFormat, OutputMetadataOptions};
+use cargo::util::{CliResult, CliError, Config};
+use cargo::util::important_paths::find_root_manifest_for_cwd;
+
+#[derive(RustcDecodable)]
+struct Options {
+    flag_output_path: OutputTo,
+    flag_manifest_path: Option<String>,
+    flag_verbose: bool,
+	flag_output_format: OutputFormat,
```

Comment:
There's a number of stylistic alignment issues with this patch throughout, could you do a pass to make sure that 4-space tabs are used consistently?